sdk/appconfiguration/azure-data-appconfiguration/src/test/java/com/azure/data/appconfiguration/ConfigurationClientTestBase.java

```diff
@@ -372,7 +372,7 @@ void listRevisionsWithMultipleLabelsRunner(String key, String label, String labe
     public abstract void listRevisionsWithRange();
 
     @Test
-    @Ignore("alzimmermsft to investigate")
+    @Ignore
```

Comment:
Filed a ticket for it instead of having the comment here - https://github.com/Azure/azure-sdk-for-java/issues/6008